s4e-backend/src/main/resources/db/migration/V4__add_resetl_password_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE password_reset (
```

Comment:
A typo in the filename. `l` after `reset`. Also, the filename should read `password_reset` and not `reset_password`.